### PR TITLE
Updating for Hugo 15

### PR DESCRIPTION
**Pull request by moxiegirl**
ping @SvenDowideit 

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: docs/windows/step_one.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ In this step, you start Docker Toolbox and run a simple Docker command.
 
     If you aren't familiar with a terminal window, here are some quick tips.
 
-    ![Teriminal](/tutimg/terminal.png)
+    {{< figure src="/tutimg/terminal.png" >}}
```
Comment (thaJeztah) [inline docs/windows/step_one.md]: think this path should be `../tutimg/terminal.png`

File: docs/windows/step_six.md [hunk 2]
```diff
@@ -46,14 +46,14 @@ If you don't already have a terminal open, open one now:
 6. Use `IMAGE ID` and the `docker tag` command to tag your `docker-whale` image.
 
     The command you type looks like this:
-    
-     ![tagger](/tutimg/tagger.png)
-    
+
+     {{< figure src="/tutimg/tagger.png" >}}
```
Comment (thaJeztah) [inline docs/windows/step_six.md]: think this path should be `../tutimg/......`

File: docs/windows/step_six.md [hunk 3]
```diff
@@ -80,25 +80,25 @@ If you don't already have a terminal open, open one now:
...
+			a82efea989f9: Image successfully pushed
+			e9e06b06e14c: Image successfully pushed
 			Digest: sha256:ad89e88beb7dc73bf55d456e2c600e0a39dd6c9500d7cd8d1025626c4b985011
-			
+
 10. Return to your profile on Docker Hub to see your new image.
 
-	 ![New image](/tutimg/new_image.png)
-	
+	 {{< figure src="/tutimg/new_image.png" >}}
```
Comment (thaJeztah) [inline docs/windows/step_six.md]: think this path should be `../tutimg/......`?

**Comment by thaJeztah** [comment]
LGTM (forgot to do that after testing :))

**Comment by moxiegirl** [comment]
author LGTM

**Pull request merged**